### PR TITLE
Limit active image pulling job parallelism in global level

### PR DESCRIPTION
**Pull request by veophi**

### CODE REVIEW

File: pkg/util/globallimiter/limiter.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
+package globallimiter
+
+import (
+	"sync"
+	"time"
+)
+
+type GlobalLimiter interface {
```
Comment (furykerry) [inline pkg/util/globallimiter/limiter.go:8]: plz give comment about what the func do and what the params means

File: pkg/controller/imagepulljob/imagepulljob_controller.go [hunk 2]
```diff
@@ -42,34 +44,48 @@ import (
...
 // and Start it when the Manager is Started.
-func Add(mgr manager.Manager) error {
+func Add(mgr manager.Manager) (err error) {
 	if !utildiscovery.DiscoverGVK(controllerKind) || !utilfeature.DefaultFeatureGate.Enabled(features.KruiseDaemon) ||
 		!utilfeature.DefaultFeatureGate.Enabled(features.ImagePullJobGate) {
 		return nil
 	}
+
+	if utilfeature.DefaultFeatureGate.Enabled(features.ImagePullJobGlobalParallelism) {
+		globalLimiter, err = initializedGlobalLimiter(20 * time.Minute)
```
Comment (furykerry) [inline pkg/controller/imagepulljob/imagepulljob_controller.go:84]: plz replace 20* min with const params, and comment about the relationship with defaultRequeueTime

File: pkg/util/globallimiter/limiter.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+		return true
+	}
+	if len(g.activeKeys) < g.limit {
+		g.activeKeys[key] = time.Now()
+		return true
+	}
+	return false
+}
+
+func (g *realGlobalLimiter) gcDeadKey(timeout time.Duration) {
```
Comment (furykerry) [inline pkg/util/globallimiter/limiter.go:51]: plz comment about why we need gc keys, it that means the pulling job can be counted in the limit for almost timeout time?

Comment (veophi) [reply pkg/util/globallimiter/limiter.go:54]: added comments.

**Pull request closed without merging**